src/main/resources/templates/fragments.html

```diff
@@ -11,33 +11,38 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond:ital,wght@0,300;0,400;1,300;1,400&family=IM+Fell+DW+Pica:ital@1&family=IM+Fell+English+SC&family=IM+Fell+Great+Primer+SC&family=Pirata+One&family=Poppins:wght@300&family=Syne&display=swap" rel="stylesheet">
-
 </head>
+
 <body>
 
 <header th:fragment="header">
-   <nav class="navbar navbar-light" style="background-color: #e3f2fd;">
-      <ul class="nav">
-          <img th:src="@{/images/treasureTask-logo.jpg}" alt="Logo Image" width="75">
-         <li class="nav-item"> <a th:href="@{/}" class="nav-link">Home</a></li>
-        <li class="nav-item"><a th:href="@{/about}" class="nav-link">About</a></li>
-         <li class="nav-item"><a class="nav-link" href="/chores">Chores</a></li>
-         <li class="nav-item"><a class="nav-link" href="/rewards">Rewards</a></li>
-         <li class="nav-item"><a class="nav-link" href="/register">Register</a></li>
-      </ul>
-      <ul class="nav" style="float:right;">
-         <li class="nav-item"><a class="nav-link" href="/logout">Log out</a></li>
-      </ul>
-   </nav>
-</header>
-<footer class="footer">
-    <nav class="navbar navbar-light" style="background-color: #e3f2fd;">
-        <div class="container text-center">
-            <span class="text-black"> Copyright &copy; TreasureTask </span>
+        <div class="d-flex flex-wrap align-items-center justify-content-center justify-content-md-between py-2">
+            <div class="col-md-3 mb-md-0 ml-3">
+                <a href="/" class="d-inline-flex link-body-emphasis text-decoration-none">
+                    <img th:src="@{/images/treasureTask-logo.jpg}" alt="Logo Image" height="75">
+                </a>
+            </div>
+
+            <ul class="nav col-12 col-md-auto mb-2 justify-content-center mb-md-0">
+                <li><a href="/chores" class="nav-link pl-3">Chores!</a></li>
+                <li><a href="/rewards" class="nav-link pl-3">Rewards</a></li>
+                <li><a href="#" class="nav-link pl-3">Crew Members</a></li>
```

Comment:
```suggestion
                <li><a href="/crew" class="nav-link pl-3">Crew Members</a></li>
```

Reply:
Let's get this in place so that when you merge with the main branch the link's set up to work properly.